tsl/test/expected/continuous_aggs_bgw.out

```diff
@@ -367,11 +368,13 @@ SELECT ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish(25);
 (1 row)
 
 SELECT * FROM sorted_bgw_log;
- msg_no | mock_time | application_name |                    msg                     
---------+-----------+------------------+--------------------------------------------
-      0 |         0 | DB Scheduler     | [TESTING] Registered new background worker
-      1 |         0 | DB Scheduler     | [TESTING] Wait until 25000, started at 0
-(2 rows)
+ msg_no | mock_time |          application_name          |                                                                 msg                                                                  
+--------+-----------+------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------
+      0 |         0 | Continuous Aggregate Policy [1001] | new materialization range for public.test_continuous_agg_table (time column time) larger than allowed in one run, truncating 12 to 2
```

Comment:
Output also different here. Previously the caggs policy didn't seem to exist here, now it does. Why? Was there previously some filtering on job_type that no longer happens?

Reply:
same as above policy output wasnt redirected to bgw_log before but now is because we dont differentiate between test jobs and normal jobs anymore